pkg/scalers/prometheus_scaler.go

```diff
@@ -150,6 +152,12 @@ func parsePrometheusMetadata(config *ScalerConfig) (meta *prometheusMetadata, er
 		return nil, fmt.Errorf("no %s given", promServerAddress)
 	}
 
+	if val, ok := config.TriggerMetadata[promServerAPIPath]; ok && val != "" {
+		meta.serverAPIPath = val
+	} else {
+		meta.serverAPIPath = "/api/v1/query"
+	}
```

Comment:
I have been checking this for a k6 extension I'm working on, and Grafana Cloud doesn't change the endpoint itself, they add the path as base. 
I mean, instead of `https://potato.com/api/v1/query` they use `https://potato.grafana.net/api/prom/api/v1/query`.

Are you sure that is enough just replacing it? Shouldn't we add it as prefix? And based on the code, I'd say that you could use Grafana Cloud endpoint and it'll be used as based url, appending this section

Reply:
Apologies I've been away. Do we have a guarantee on what the correct endpoint is from Grafana? If you are correct then the PR will need updating :)

Reply:
Also, I might of missed this but how do I sign the commit?

Reply:
@SubatomicHero can you try following [this](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#i-didnt-sign-my-commit-now-what) for signing the commit?

Reply:
I discovered it meanwhile I was working on this: https://github.com/JorTurFer/xk6-input-prometheus
It uses `github.com/prometheus/client_golang` which always append the `/api/v1/query` to all the query requests. We are using it to request metrics from Grafana Cloud on performance tests and it works. I think that the scaler already can work with Grafana Cloud just setting the url that Grafana shows on the portal (because it's already appended)
https://github.com/kedacore/keda/blob/b18ccd6a10f9b626ea41fb6927593d0110612de1/pkg/scalers/prometheus_scaler.go#L270

I'll check later on if the scaler already worked with Grafana Cloud

Reply:
Sorry for the long delay. I can confirm that it works with Grafana Cloud just adding the path in the metadata section without any change in the code.
I have testes with this values and it has worked correctly:
![image](https://github.com/kedacore/keda/assets/36899226/aa31f755-c7f3-4153-b16c-25623f12312d)

I guess that we can close the PR and the issue as well. WDYT?

Reply:
LGTM